cleverpush/src/main/java/com/cleverpush/CleverPush.java

```diff
@@ -460,7 +460,8 @@ public SessionListener initSessionListener() {
                     this.initFeatures();
                 }
 
-                if (this.appBannerModule != null) {
+                Logger.d("initSession", "appBannerModule" + this.getCurrentActivity());
```

Comment:
remove log

Reply:
`    if (this.appBannerModule != null && this.getCurrentActivity() != null) {
                    this.appBannerModule.initSession(channelId);
                } else {
                    Logger.e("LOG_TAG", "activity null: " + getCurrentActivity());
                }`

Reply:
is it okay?